### ISSUE TITLE
[Request] New color schemes from Vice 3.7

### ISSUE THREAD

**Issue by ted210** [randyrossi/bmc64#231]
Would it be possible to incorporate the updated color schemes that were included in Vice 3.7?

The new color scheme is more accurate as it was obtained by measurement (rather than the previous estimated values) and as a result the colors are generally more saturated than the previous scheme.

**Comment by WebbedQuasar** [comment]
I don't use VICE on my PC, but I've been using the Colodore palette since implementing in BMC64 version 3.7.
The Colodore palette is based on measurements and irons out the incorrect measurements of the previous Pepto palette. The Colodore palette comes very close to the colours of a real C64, a complete 100% correct C64 palette is not possible with emulator technology, only approximations. I'm very happy with Colodore, so I would recommend using this one.

**Comment by ted210** [comment]
Thanks, that's useful information and good enough for me. I'll close the request.

**Comment by rhester72** [comment]
Which palette was being referred to, exactly?  Pixcen?  (That's the only one I can see that's missing.)

**Comment by ted210** [comment]
Perhaps I didn't describe this as precisely as I should have.  I was referring to this text in the changelog:

** C64 fixes
------------

- Use measured PAL color vectors (provided by Tobias)

**Comment by rhester72** [comment]
OK, I get it now.  I originally thought this was a palette thing - Pixcen is new-ish, TheC64 was just added, but neither are particularly interesting.

What they were really referring to was this:

- CRT emulation improvements:
  - uses measured (different) color coordinates for odd/even lines when possible

which looks like this in source form (very incomplete, just conceptual):

https://sourceforge.net/p/vice-emu/code/43362/log/?path=/trunk/vice/src/vicii/vicii-color.c

The short version - so much change has occurred in VICE in this regard that backporting it all into BMC64 would be quite a lift.  I don't think it's impossible, but it is daunting, and tbh I personally don't have the time to take on such an effort in the foreseeable future.  Leaving this documentation here in case someone would like to revisit in the future (who knows, maybe me).
